AppVeylor NuPIC test failure after Merge pull request #2950

NuPIC test failure https://ci.appveyor.com/project/numenta-ci/nupic/branch/master (after Merge pull request #2950 from subutai/2902)   
```
tests.integration.nupic.opf.hotgym_regression_test.HotgymRegressionTest testMethod=testHotgymRegression

        # Changes that affect prediction results will cause this test to fail.
        # If the change is understood and reviewers agree that there has not been a
        # regression then this value can be updated to reflect the new result.
>      self.assertAlmostEqual(float(lastRow[14]), 5.90725098207)
E      AssertionError: 5.87970891453 != 5.90725098207 within 7 places
```

--- a/.nupic_modules
+++ b/.nupic_modules
@@ -1,3 +1,3 @@
 # Default nupic.core dependencies (override in optional .nupic_config)
 NUPIC_CORE_REMOTE = 'git://github.com/numenta/nupic.core.git'
-NUPIC_CORE_COMMITISH = '621a1b7fb3d0ff4ff3d2358a782095350945c266'
+NUPIC_CORE_COMMITISH = '7de0c5992364dc847dc2b8303093ff8aea52b61e'

--- a/tests/integration/nupic/opf/hotgym_regression_test.py
+++ b/tests/integration/nupic/opf/hotgym_regression_test.py
@@ -61,8 +61,8 @@ class HotgymRegressionTest(unittest.TestCase):
                          "steps=1:window=1000:field=consumption")
         lastRow = collections.deque(reader, 1)[0]
 
-      # Changes that affect prediction results will cause this test to fail.
-      # If the change is understood and reviewers agree that there has not been a
+      # Changes that affect prediction results will cause this test to fail. If
+      # the change is understood and reviewers agree that there has not been a
       # regression then this value can be updated to reflect the new result.
       self.assertAlmostEqual(float(lastRow[14]), 5.90725098207)
 

--- a/tests/integration/nupic/opf/hotgym_regression_test.py
+++ b/tests/integration/nupic/opf/hotgym_regression_test.py
@@ -63,7 +63,7 @@ class HotgymRegressionTest(unittest.TestCase):
       # Changes that affect prediction results will cause this test to fail. If
       # the change is understood and reviewers agree that there has not been a
       # regression then this value can be updated to reflect the new result.
-      self.assertLess(abs(float(lastRow[14])-5.90725098207), 0.04)
+      self.assertAlmostEqual(float(lastRow[14]), 5.87970891453)
 
     finally:
       shutil.rmtree(resultsDir, ignore_errors=True)

--- a/tests/integration/nupic/opf/hotgym_regression_test.py
+++ b/tests/integration/nupic/opf/hotgym_regression_test.py
@@ -28,7 +28,6 @@ explicitly updated to match the new results.
 import collections
 import csv
 import os
-import pkg_resources
 import shutil
 import unittest
 
@@ -64,7 +63,7 @@ class HotgymRegressionTest(unittest.TestCase):
       # Changes that affect prediction results will cause this test to fail. If
       # the change is understood and reviewers agree that there has not been a
       # regression then this value can be updated to reflect the new result.
-      self.assertAlmostEqual(float(lastRow[14]), 5.90725098207)
+      self.assertLess(abs(float(lastRow[14])-5.90725098207), 0.04)
 
     finally:
       shutil.rmtree(resultsDir, ignore_errors=True)

